Packs/CrowdStrikeFalcon/Scripts/CsFalconLiftHostContainment/README.md

```diff
@@ -0,0 +1,24 @@
+Lifts containment on the host, which returns its network communications to normal.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | Utilities |
+| Cortex XSOAR Version | 6.1.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| ids | A comma-separated list of host agent IDs \(AIDs\) of the hosts to contain. Get an agent ID from a detection. |
```

Comment:
A comma-separated list of host agent IDs \(AIDs\) to contain. Get agent IDs from detections.